tests/UnifiedSpecTests/Util.php

```diff
@@ -26,13 +34,113 @@
 
 final class Util
 {
+    /**
+     * Array to fill, which contains the schema of allowed attributes for operations.
+     */
+    private static $args = [
+        Operation::OBJECT_TEST_RUNNER => [
+            'assertCollectionExists' => ['databaseName', 'collectionName'],
+            'assertCollectionNotExists' => ['databaseName', 'collectionName'],
+            'assertIndexExists' => ['databaseName', 'collectionName', 'indexName'],
+            'assertIndexNotExists' => ['databaseName', 'collectionName', 'indexName'],
+            'assertSameLsidOnLastTwoCommands' => ['client'],
+            'assertDifferentLsidOnLastTwoCommands' => ['client'],
+            'assertNumberConnectionsCheckedOut' => ['connections'],
+            'assertSessionDirty' => ['session'],
+            'assertSessionNotDirty' => ['session'],
+            'assertSessionPinned' => ['session'],
+            'assertSessionTransactionState' => ['session', 'state'],
+            'assertSessionUnpinned' => ['session'],
+            'failPoint' => ['client', 'failPoint'],
+            'targetedFailPoint' => ['session', 'failPoint'],
+            'loop' => ['operations', 'storeErrorsAsEntity', 'storeFailuresAsEntity', 'storeSuccessesAsEntity', 'storeIterationsAsEntity'],
+        ],
+        Client::class => [
+            'createChangeStream' => ['pipeline', 'typeMap', 'session', 'fullDocument', 'resumeAfter', 'startAfter', 'startAtOperationTime', 'batchSize', 'collation', 'maxAwaitTimeMS'],
```

Comment:
`typeMap` can be removed from everything. That is a PHP-specific option and will never appear in any spec tests.